utilities/opentelemetry_telemetry/lib/opentelemetry_telemetry.ex

```diff
@@ -44,11 +44,11 @@ defmodule OpentelemetryTelemetry do
 
     def handle_event(_event,
                 %{duration: duration},
-                %{kind: kind, reason: reason, stacktrace: stacktrace} = metadata,
+                %{kind: _kind, reason: reason, stacktrace: stacktrace} = metadata,
                 %{type: :exception, tracer_id: tracer_id}) do
-        ctx = OpentelemetryTelemetry.set_current_telemetry_span(tracer_id, metadata),
-        status = Opentelemetry.status(:error, to_string(reason, :utf8))
-        OpenTelemetry.Span.record_exception(ctx, kind, reason, stacktrace, [duration: duration])
+        ctx = OpentelemetryTelemetry.set_current_telemetry_span(tracer_id, metadata)
+        status = OpenTelemetry.status(:error, to_string(reason))
+        OpenTelemetry.Span.record_exception(ctx, reason, stacktrace, [duration: duration])
```

Comment:
```suggestion
        OpenTelemetry.Span.record_exception(ctx, reason, stacktrace)
```

@tsloughter I just realized, should we setting the duration based on the `:telemetry` or should we let OTEL handle this? I haven't set the duration manually ever, so now I am curios to see what should be the "best practices" here

None of the other instrumentation libraries do it: Looking at opentelemetry_grpc, opentelemetry_phoenix, opentelemetry_broadway, opentelemetry_nebulex, opentelemetry_oban/job_handler - they all ignore the measurements parameter in exception handlers.

Reply:
I find that odd too. Unless there is something about a duration attribute for exceptions in the semconv we shouldn't have that.